packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-point.tsx

```diff
@@ -26,6 +26,7 @@ type Props = {
     sequenceNumber?: number;
     onBlur?: (event: React.FocusEvent) => unknown;
     onClick?: () => unknown;
+    onDragEnd?: () => unknown;
```

Comment:
I'm not the biggest fan of adding a new callback, but it seemed like the best way to handle this situation. I'm open to suggestions!